### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by lucaslovato**
- [X] I have searched the [PR](https://github.com/Marketing-Automation-Systems/jellyfish/pulls/) of this repository and believe that this is not a duplicate.

**Describe the PR**

This PR adds a Pull Request template for the project

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+ [ ] I have searched the [PR](https://github.com/Marketing-Automation-Systems/jellyfish/pulls/) of this repository and believe that this is not a duplicate.
+
+<!-- Checked checkbox should look like this: [x] -->
+
+**Describe the PR**
+
+This PR closes [#ISSUE_NUMBER](ISSUE LINK)
```
Comment (leonardo-brickabode) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: You don't need to use [#ISSUE_NUMBER](ISSUE LINK), just #ISSUE_number and github will autocomplete for you. For instance, `Closes #13`

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<!--
+  Thank you very much for contributing to the Jellyfish team by creating an issue! ❤️
+  To avoid duplicate PR we ask you to check off the following list.
+-->
+
+
+
+<!-- Short bullet list with the new changes proposed-->
+
+**To Reproduce what was done**
```
Comment (leonardo-brickabode) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Maybe we should include a summary of what was done.

Comment (lucaslovato) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: This is supposed to be done on line 9

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (lucaslovato) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Yes I know, I tried to make more explicit, but I'll change.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+**Describe the PR**
+
+This PR closes #ISSUE_NUMBER
+
+<!-- Provide a description of the changes proposed in the pull request -->
+
+<!--
+  Thank you very much for contributing to the Jellyfish team by creating an issue! ❤️
+  To avoid duplicate PR we ask you to check off the following list.
+-->
```
Comment (leonardo-brickabode) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Maybe move this to the top of the file? I feel like this part is out of context here.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+ [ ] I have searched the [PR](https://github.com/Marketing-Automation-Systems/jellyfish/pulls/) of this repository and believe that this is not a duplicate.
+
+<!-- Checked checkbox should look like this: [x] -->
+
+**Describe the PR**
+
```
Comment (leonardo-brickabode) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Maybe give an example here
```suggestion
<!-- For instance: Closes #10  -->
```

Comment (lucaslovato) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: I don't see purpose to make the example as comment, so I left this phrase on the PR.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 3 above)
Comment (lucaslovato) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Changed.

**Pull request merged**